app/helpers/bill_parser.rb

```diff
@@ -0,0 +1,40 @@
+	module BillConverter
+
+		def parse_bill
+			bill_source = APP_ROOT.to_path+'/public/uploads/'+params[:bill][:name]
+
+			File.open('public/uploads/'+params[:bill][:name], "w") do |f|
+				f.write(params[:bill][:tempfile].read)
```

Comment:
This method is called `parse_bill` which implies it is only reading and parsing something. This method also writes to a file. The name should inform the reader of this.